Documentation/observability/visibility.rst

```diff
@@ -66,6 +66,26 @@ In order for Cilium to populate the ``INGRESS ENFORCEMENT``, ``EGRESS ENFORCEMEN
 and ``VISIBILITY POLICY`` fields, it must run with ``--endpoint-status=policy``
 to make field values visible.
 
+Security Implications
+---------------------
+
+Monitoring Layer 7 traffic comes with some serious security considerations in
+terms of handling potentially sensitive information such as usernames,
+passwords, query parameters, API keys, etc.
+
+.. warning::
+
+    By default, Hubble does **not** protect potentially sensitive information
```

Comment:
nit: "protect" is vague here, maybe redact or edit or hide would be more appropriate?